modules/liveIntentIdSystem.js

```diff
@@ -227,6 +228,10 @@ export const liveIntentIdSubmodule = {
         result.sovrn = { 'id': value.sovrn, ext: { provider: LI_PROVIDER_DOMAIN } }
       }
 
+      if (value.thetradedesk) {
+        result.thetradedesk = { 'id': value.thetradedesk, ext: { provider: getRefererInfo().domain || LI_PROVIDER_DOMAIN } }
```

Comment:
Can we test that without injecting the value?

Reply:
without mocking the refererInfo, the domain value would be: `localhost:9876` 
I am trying to find a way to change the page&location in the test to have a proper domain.

Reply:
as discussed, will test it in a page.

Reply:
Agree. This is the correct provider :)